Sources/TuistGenerator/Mappers/AutogeneratedSchemesProjectMapper.swift

```diff
@@ -200,6 +200,19 @@ public final class AutogeneratedSchemesProjectMapper: ProjectMapping { // swiftl
         )
     }
 
+    private func defaultArguments(for targets: [Target]) -> Arguments? {
```

Comment:
To avoid duplication, I would either remove the helper taking a single target, or I would reuse it inside this one

Reply:
I was actually reusing the existing one first, then I changed this use `TestableTarget` instead of `Target` and then I changed it back and forgot to reuse again. 
Fixed.